### ISSUE TITLE
task

### ISSUE THREAD

**Issue by Restuta** [Restuta/rcn.io#17]

**Comment by Restuta** [comment]
USAC official https://m.facebook.com/tim.burgess.56

**Comment by Restuta** [comment]
https://m.facebook.com/velophil

Have concerns that MTB races are not currently accessible/discoverable (they may be non-usac ones, [see this FB thread](https://m.facebook.com/groups/31318051584?view=permalink&id=10153714089526585))

**Comment by Me4ta** [comment]
We need to make a cards and give them to everyone on races.

**Comment by Restuta** [comment]
@me4ta yes, but this is for "after Pblic MVP", e.g. "Official release 1.0"

**Comment by Restuta** [comment]
Add the LKHC

**Comment by Restuta** [comment]
Review http://www.sfsprints.com/#!the-race/caio

**Comment by Restuta** [comment]
Related to NCNCA (need tofigure out his role) https://m.facebook.com/jim.dosenbach

**Comment by Restuta** [comment]
Add to list:
https://m.facebook.com/steve.rosefield Red Kite organizer

**Comment by Restuta** [comment]
Wccc http://www.wcccycling.org/schedule.html

**Comment by Restuta** [comment]
Clubs http://www.ncnca.org/sites/default/files/internal/clubs/2016NCNCAClubs.pdf

**Comment by Restuta** [comment]
NCNCA Photographers

**Comment by jpjoyal** [comment]
Hello! 

My name and email shows up in google because of this issue entry. Can you amend it to remove this private information. Thank you
